functional-tests/resources/redact_and_replace_payload.json

```diff
@@ -0,0 +1,27 @@
+{
+  "text": "hello world, my name is Jane Doe. My number is: 034453334",
+  "transformations": {
+    "NAME": {
+      "type": "redact",
+      "new_value": "ANONYMIZED"
+    },
+    "PHONE_NUMBER": {
+      "type": "replace",
+      "new_value": ""
```

Comment:
Is the replace transformation adding anything for this scenario not covered by others?

Reply:
An E2E for it.
It will move to your functional tests in the future.

Reply:
Sorry, missed that its both for replace and reduct, I would split to get a more precise indication in case one will fail, and it will reduce the payloads to be easier to read.

Reply:
ok